lib/rubygems/executor.rb

```diff
@@ -0,0 +1,124 @@
+# frozen_string_literal: true
+require 'open-uri'
+require 'json'
+require 'rubygems/platform'
+
+module Gem
+  module Web
+    class Executor
+
+      def open_default_browser_cmd(local_os, version, uri)
```

Comment:
This would be really hard to maintain, would be better if it reads the `browser` from an environment variable

Reply:
@bronzdoc That's a nice option. `gem-web` used the gem `Launchy` to open links, so it was kinda hard to think a good way to reproduce this.